langevals/evaluators/langevals/tests/test_exact_match.py

```
@@ -133,3 +135,60 @@ def test_langeval_exact_match_numbers_not_match():
+    """JS-loose layer must not short-circuit the existing trim/punct/case chain
+    when both sides are free-form text."""
+    entry = ExactMatchEntry(
+        output="  Hello!  ",
+        expected_output="hello",
+    )
+    settings = ExactMatchSettings(
+        case_sensitive=False, trim_whitespace=True, remove_punctuation=True
+    )
+    evaluator = ExactMatchEvaluator(settings=settings)
+    result = evaluator.evaluate(entry)
+    assert result.passed == True
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `result.passed:` for truth checks

Replace with `result.passed`

(E712)